cogs/sudo.py

```diff
@@ -86,33 +94,105 @@ async def _reload(self, ctx: Context, *, extension: str) -> None:
         else:
             await ctx.send("\N{SQUARED OK}")
 
+    @sudo.command()
+    @check(is_owner)
+    async def unload(self, ctx: Context, *, extension: str) -> None:
+        """Unloads a module."""
+        try:
+            self.bot.unload_extension(f"cogs.{extension}")
+        except Exception:
+            await ctx.send(f"```py\n{traceback.format_exc()}\n```")
+        else:
+            await ctx.send("\N{SQUARED OK}")
+
     @sudo.command()
     @check(is_owner)
     async def restart(self, ctx: Context) -> None:
         """Restart The bot."""
+        await ctx.message.add_reaction("✅")
         await self.bot.logout()
-        os.system("python main.py")
+        time.sleep(2)
+        os.system("pipenv run start main.py")
+
+    @sudo.command()
+    @check(is_owner)
+    async def createinv(self, ctx, channelid: int) -> None:
+        channel = self.bot.get_channel(channelid)
+        InviteURL = await channel.create_invite(max_uses=1)
+        await ctx.send(InviteURL)
+
+    @group(hidden=True)
+    async def botstatus(self, ctx: Context) -> None:
+        """
+        Change the status of the bot
+
+        `botstatus playing <new status>` - Change playing status
+        `botstatus watching <new status>` - Change watching status
+        `botstatus listening <new status>` - Change listening status
+        """
+        if ctx.invoked_subcommand is None:
+            pass
```

Comment:
Consider making this a single command which accepts `type` of `t.Literal["playing", "listening", "watching"]`, there is no point in having 3 group subcommands which does exactly the same thing with a single difference